app/src/main/java/com/example/mytestproject/mapp/Mapp.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.mytestproject.mapp
+
+import com.example.mytestproject.data.repository.responseWeatherData.WeatherDataApi
+import com.example.mytestproject.ui.models.weatherDataModel.WeatherData
+
+fun WeatherDataApi.mapToWeatherData(): WeatherData {
+    return WeatherData(city_name = this.city_name, data = this.data)
```

Comment:
Let's create a separate class for this.
There is a good article about it:
https://proandroiddev.com/the-real-repository-pattern-in-android-efba8662b754